service/collector.go

```diff
@@ -215,17 +217,30 @@ func (col *Collector) shutdown(ctx context.Context) error {
 		errs = multierr.Append(errs, fmt.Errorf("failed to shutdown config provider: %w", err))
 	}
 
+	if err := col.shutdownServiceAndTelemetry(ctx); err != nil {
+		errs = multierr.Append(errs, err)
+	}
```

Comment:
when using `multierr.Append` unless you want to enhance the err and wrap it with more context (like using `fmt.Errorf`, you can simply do this:

```suggestion
	errs = multierr.Append(errs, col.shutdownServiceAndTelemetry(ctx))
```

Reply:
Thanks for the suggestion. I'm not using to using this package so forgot it's ok to pass in a nil error.